app/Http/Controllers/FileController.php

```diff
@@ -25,7 +25,7 @@ public function get(Request $request, File $file)
         $this->authorize('view', $file);
 
         if (Storage::exists($file->full_path)) {
-            return response()->file(Storage::path($file->full_path), ['Content-Disposition' => 'inline; filename="' . $file->name . '"', 'Content-Type' => Storage::mimeType($file->name)]);
+            return response()->file(Storage::path($file->full_path), ['Content-Disposition' => 'inline; filename="' . $file->name . '"', 'Content-Type' => Storage::mimeType($file->full_path)]);
```

Comment:
@blt950 For some reason, this line returns false even in Dockerized version, but it does work properly when developing in Docker, but not on production without Docker ...

Reply:
Could you please rephrase your comment? I'm not sure I understand what you're trying to say
![image](https://github.com/Vatsim-Scandinavia/controlcenter/assets/2505044/3332de04-b0c5-4b71-9871-d79d30612457)

Reply:
Sure, sorry for my English.

Before change:

`Storage::mimeType($file->name)` returned false in Docker and in non-Docker environment. But when user opened link to download the file, it worked on Dockerized version anyway but NOT on non-dockerized version. I dunno why, might be configuration of server which might differ between Docker and ours and Sarantos's server

After change:

It works everywhere.

Reply:
Got'cha! Well, seems like a typo from 2 years back that we try setting the content type based on the name of the file instead of path (lol wat). This fix seems to work on my environment, so all good.